bootloader/bootloader.go

```diff
@@ -343,3 +343,40 @@ func ForGadget(gadgetDir, rootDir string, opts *Options) (Bootloader, error) {
 	}
 	return nil, ErrBootloader
 }
+
+// BootImage represents each image in the chains of trusted assets and kernels
+// used in the boot process. A boot image can be an EFI binary or a snap file
+// containing an EFI binary.
```

Comment:
image is very much a EFI term,  we should't talk about EFI here.  I think this should be just called BootFile and mention something about a file participating and loaded in the boot process.

Reply:
Sorry, this was intended to be called BootFile since the beginning but the wrong name was used in the structure definition and then reused everywhere.